### PR TITLE
Remove `beacon.js`

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Remove `beacon.js`. The `postJson` method isn’t used anywhere, and the analytics for ad rendered are not being looked at.

The @guardian/commercial-dev team is not using these analytics currently, and is currently building a new system for monitoring ad speed.

The Ophan team is not receiving this data either:

> Hi, thanks for the cc- however, I don't think this code is related to Ophan - ie it's not used for writing data into Ophan's Tracker at ophan.theguardian.com - I think it's actually for monitoring ad-rendering, if I'm reading #17854 (files) correctly? (@rtyley [on #23702](https://github.com/guardian/frontend/pull/23702#discussion_r613363458))

Introduced in #17854 and removed in #21652.

## What is the value of this and can you measure success?

Shed dead/unused code.

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by arelra — APPROVED** [review]
🎉

File: static/src/javascripts/projects/common/modules/analytics/beacon.js [hunk 1]
```diff
@@ -1,27 +0,0 @@
-import config from 'lib/config';
-
-const fire = (path) => {
-    const img = new Image();
-
-    img.src = config.get('page.beaconUrl') + path;
```
Comment (ioanna0) [inline static/src/javascripts/projects/common/modules/analytics/beacon.js:6]: should we also remove `beaconUrl` from page configuration?

**Comment by rtyley** [comment]
> Introduced in #17854

...having looked a bit more, I think the history of this code goes back even further to PR https://github.com/guardian/frontend/pull/2562 in December 2013 (_"Added beacon when analytics call is finished... so we can independently confirm our analytics are still working"_)!

Well done on deleting some very old code!

File: static/src/javascripts/projects/common/modules/analytics/beacon.js (hunk 1 above)
Comment (mxdvl) [reply static/src/javascripts/projects/common/modules/analytics/beacon.js:6]: I’m not entirely sure we can, it’s still used in DCR in [several places][beaconUrl], including for Google Analytics:

https://github.com/guardian/dotcom-rendering/blob/44bf079a459f5c29a5c3a03d946bbd28dd5df65a/src/web/browser/ga/ga.ts#L155

[beaconUrl]: https://github.com/guardian/dotcom-rendering/search?q=beaconurl

**Comment by mxdvl** [comment]
Merged into #23702.

**Pull request merged**